brave/browser-laptop
# Refactor findbar with Aphrodite

Closes #9233

- Apply BrowserButton to the close button
- Apply Textbox to the input element
- Add color styles to theme.js

Auditors: @cezaraugusto

Test Plan:
1. Run `npm run test -- --grep='findbar'`
2. Open facebook.com
3. Display find bar
4. Make sure the bar is displayed properly

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
